samples/cmaf-ham-conversion/src/index.ts

```diff
@@ -27,12 +28,12 @@ setHlsParser((text: string) => {
 
 	parser.push(text);
 	parser.end();
-	const parsedHlsManifest = parser.manifest;
+	const parsedHlsManifest = parser.manifest as any;
```

Comment:
(Optional) I suggest avoiding `as any` where possible

Reply:
The only alternative that I know is `as unknown`. Is that any better?